tlovett1/safe-redirect-manager
# PHP 8.1: Ensure PHPCS doesn't warn on the now harmless ini_set

### Description of the Change

While #327 resolved a deprecation warning, the code is still flagged by PHPCS as being problematic. This change builds on #327 to add the `phpcs:ignore` statement so that the code doesn't hold up any processes unnecessarily by flagging something that's no longer an issue. 

Closes #283 

### How to test the Change

1. Run PHPCS using PHP 8.1 (e.g. with `<config name="testVersion" value="8.1" />` in phpcs.xml)
2. Observe that PHPCS produces a warning for inc/functions.php:415
3. Apply this patch and re-run PHPCS, there will be no warning

### Changelog Entry
> Added - phpcs:ignore on the now safe `ini_set()`

### Credits
Props @philipjohn

### Checklist:
